### DISCUSSION TITLE
Running on Pi Zero W with 240x240 display

### DISCUSSION THREAD

**Discussion by babraham76** [taoteh1221/Slideshow_Crypto_Ticker#13] [Show and tell]
Just wanted to show you this runs great on my Pi Zero W with a 240x240 display.  I leave it running on my desk for a quick glance at the BTC price at Coinbase... Thanks!!!

![IMG_2031](https://user-images.githubusercontent.com/42757818/112915879-a1ef5480-90bc-11eb-8dc3-4c82f0969919.JPG)

**Comment by taoteh1221** [comment]
Looks great, thanks for the feedback! :)

I have a bug fix release coming out end of this week, as the other exchange APIs try to connect a websocket (in the background) even if you have their markets turned off. That will be fixed (along with a few other tweaks).

Enjoy the ticker!

**Comment by banshee1971** [comment]
Wow ! i want that too !

**Comment by onyxcinema** [comment]
Were you able to get this up and running without doing any Python trickery? Had a hard time trying to set up an ePaper display for the ticker yesterday. Gonna try this same adafruit display next.

**Reply by taoteh1221** [reply]
@onyxcinema Running this command over SSH or a terminal app auto-installs the ticker, no manual setup required:

```
wget --no-cache -O TICKER-INSTALL.bash https://git.io/Jqzjk;chmod +x TICKER-INSTALL.bash;sudo ./TICKER-INSTALL.bash
```

**Reply by onyxcinema** [reply]
Oh yeah I got it working over HDMI but was wondering if it just as simple with the tft 240x240 SPI interface display in the above post.

**Reply by taoteh1221** [reply]
Cool. I personally have no idea on a zero w with a super-tiny screen, but as long as you can get the desktop GUI to show on the screen you should be all set, as it requires running the desktop.

**Reply by taoteh1221** [reply]
Technically really only requires opening index.html in any modern browser, but as far as the scripting automation built-in goes, it requires the desktop.

**Comment by onyxcinema** [comment]
@babraham76 How did you get this to work on the adafruit display? I just received the exact same 240x240 display and installed their kernal driver and installed the ticker but it didn't work :-(
